### DISCUSSION TITLE
Filter data in datagrid via URL

### DISCUSSION THREAD

**Discussion by elenaaralla** [serenity-is/Serenity#7051] [Help]
Hello,

I would like to filter data via URL; 

For example, if I click this link:

<My app base url>/Common/Document?**CompanyId**=99bb0232-dfc2-41c8-b6a4-e22b3cdbe3a3

I would like to see quick filter filled with  **CompanyId** passed on URL and grid rows filtered by this value (see image)

How can I perform this? Is there an example on documentation? Where?

Thank you,
Elena.

![Schermata 2023-12-30 alle 12 25 58](https://github.com/serenity-is/Serenity/assets/14870650/3737961a-d9c6-4dc9-9b8a-7d5aa190271d)

P.S.: I wish you an Happy new year!

**Comment by luongnv6** [comment]
https://demo.serenity.is/Northwind/Product?cat=1
